docs/references/concepts.md

```diff
@@ -4,16 +4,35 @@ description: Fundamental user-facing concepts
 
 # Key Concepts
 
+## Instance
+
+An "instance" of Opstrace runs inside your cloud account.
+It is installed with the CLI `opstrace create` command.
+This name must be _globally_ unique and limited to 23 characters (`[a-z0-9-_]`) because it used to provide a conveient DNS name:  `<instance_name>.opstrace.io`.
+Once chosen, an instance name cannot be changed.
+
 ## Tenants
 
-The concept of multitenancy allows for isolating environments within the same Opstrace cluster.
-Tenants are logically isolated, but otherwise share the same \(physical\) base system.
+A "tenant" is a unit of isolation.
+[Multi-tenant systems](https://en.wikipedia.org/wiki/Multitenancy) allow for logical separation of concerns, while otherwise sharing the same underlying system.
+
+Opstrace instances support multiple, secured tenants.
```

Comment:
I have always wondered what the value is of saying ", secured" here. To me, this is marketing speech. 

Rationally, the following sentence has good and _important_ information content:

> A single Opstrace instance supports multiple tenants

The main task of this sentence is to point out the one-to-many relationship here.

To make this message unambiguous, it is important to use `Opstrace instance" in singular.

If we want to say something about the relationship between tenants and security let's do this in its own sentence or paragraph!

Reply:
For now, I will remove the ", secured".

A general term such as "secure" highlights our value proposition without being wordy and slowing the reader down.